framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInference.java

```diff
@@ -44,91 +54,112 @@ void updateInferredConstructorParameterTypes(
      * Updates the parameter types of the method {@code methodElt} based on the arguments in the
      * method invocation {@code methodInvNode}.
      *
+     * <p>For each method parameter in methodElt:
```

Comment:
This comment should be explicit about whether this also updates the receiver parameter (I think it does).